.ci/targets/android_platform_tests.yaml

```diff
@@ -5,23 +5,23 @@ tasks:
   - name: download Dart and Android deps
     script: script/tool_runner.sh
     infra_step: true
-    args: ["fetch-deps", "--android", "--supporting-target-platforms-only"]
+    args: ["fetch-deps", "--android", "--supporting-target-platforms-only", "--exclude=script/configs/still_requires_api_33_avd.yaml"]
```

Comment:
The changes to this file need to wait for the second PR; moving them now would effectively disable CI testing for the plugins in this file.